src/axom/inlet/LuaReader.hpp

```diff
@@ -38,15 +39,9 @@ namespace inlet
 class LuaReader : public Reader
 {
 public:
-  /*!
-   *****************************************************************************
-   * \brief Destructor for the LuaReader.
-   *
-   * This performs any cleanup work the Reader needs to do before going
-   * away.
-   *****************************************************************************
-   */
-  ~LuaReader();
+  LuaReader() {
+    m_lua.open_libraries(sol::lib::base);
```

Comment:
Does this need to be guarded, e.g. with an error message if it cannot open the library, or does SOL/lua already handle that for you?

Reply:
It's not needed for our usage of Lua, so I've removed it.